code/game/objects/effects/effects.dm

```diff
@@ -1,9 +1,41 @@
 
 //objects in /obj/effect should never be things that are attackable, use obj/structure instead.
 //Effects are mostly temporary visual effects like sparks, smoke, as well as decals, etc...
-
 /obj/effect
 	icon = 'icons/effects/effects.dmi'
-	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
```

Comment:
/obj/effect, along with /obj/screen and /obj/docking_port (which should probable be an effect), are the only /obj/ not to follow the damage system. So we override the damage related procs so you can't punch a bloodstain to destroy it.